README.md

```diff
@@ -324,14 +333,24 @@ On the machine you are deploying from:
 2. Install Latest Nix with GPG Verification
 
 	```
-	curl -o install-nix-2.1.3 https://nixos.org/nix/install
-	curl -o install-nix-2.1.3.sig https://nixos.org/nix/install.sig
+	curl -o install-nix-2.2.1 https://nixos.org/nix/install
+	curl -o install-nix-2.2.1.sig https://nixos.org/nix/install.sig
 	gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
-	gpg2 --verify ./install-nix-2.1.3.sig
-	sh ./install-nix-2.1.3
+	gpg2 --verify ./install-nix-2.2.1.sig
+	sh ./install-nix-2.2.1
 	. /home/user/.nix-profile/etc/profile.d/nix.sh
 	```
 
+	Sandboxing is enabled by default on Nix <2.2. This may cause problems on your system. If you get an error similar to
+
+	```
+	error: cloning builder process: Operation not permitted
+	error: unable to start build process
+	/tmp/nix-binary-tarball-unpack.hqawN4uSPr/unpack/nix-2.2.1-x86_64-linux/install: unable to install Nix into your default profile
+	```
+
+	try disabling sandboxing by adding `sandbox = false` to `/etc/nix/nix.conf`.
```

Comment:
I haven't tested nix 2.2 but apparently this happens when nix isn't installed as root because sandboxing needs root permissions therefore imo better to specify that this happens because nix isn't isntalled as root and that alternatively you can install as root or build as user (as recommended here https://nixos.org/nix/manual/#conf-sandbox).

Reply:
Fixed in 36b37eb

Reply:
only fixed in one of the two locations

Reply:
Fixed in a7eb091. Will be more careful next time.